src/components/Hero/Hero.css

```diff
@@ -0,0 +1,45 @@
+.main {
+  width: 1280px;
```

Comment:
bad practice. set in terms of %. Will create major issues in screen with different pixel density